packages/shared/src/components/cards/Freeform/FreeformList.tsx

```diff
@@ -98,9 +99,11 @@ export const FreeformList = forwardRef(function SharePostCard(
               onShare={onShare}
               post={post}
               imageProps={{
-                src: image,
-                className: 'my-2 mobileXXL:self-start w-full',
                 alt: 'Post Cover image',
+                className: 'my-2 mobileXXL:self-start w-full',
+                fetchPriority: eagerLoadImage ? 'high' : 'auto',
```

Comment:
I would use the lowercased fetchpriority and add a @ts-expect-error - Not supported by React.

The good thing about expect error as opposed to ignore is that if the line below ever stops giving an error, the line itself will error to alert you to remove it, so it won't get stuck around forever after it's irrelevant.